device_calendar/android/src/main/kotlin/com/builttoroam/devicecalendar/DeviceCalendarPlugin.kt

```diff
@@ -59,15 +64,28 @@ class DeviceCalendarPlugin() : MethodCallHandler {
 
         when (call.method) {
             RETRIEVE_CALENDARS_METHOD -> {
-                val calendars = _calendarService.retrieveCalendars();
+                _calendarService.retrieveCalendars();
             }
             RETRIEVE_CALENDAR_EVENTS_METHOD -> {
                 val calendarId = call.argument<String>(CALENDAR_ID_ARGUMENT);
-                if (calendarId?.isNullOrEmpty() ?: true) {
+                if (calendarId == null || calendarId.isEmpty()) {
```

Comment:
you probably noted it already but will need to handle reading in the start and end date arguments with the changes i recently made

Reply:
Yep